README.md

```diff
@@ -183,6 +185,25 @@ export interface ThrottlerStorage {
 
 So long as the Storage service implements this interface, it should be usable by the `ThrottlerGuard`.
 
+## Working with Websockets
+
+To get the `ThrottlerModule` working with websockets, not much is needed besides regular
+configuration. The things to make note of are that 1) you cannot bind the guard with
+`APP_GUARD` or `app.useGlobalGuards()` due to how Nest binds global guards and 2) when a limit is
+reached, Nest will emit an `exception` event, so make sure there is a listener ready for this.
+Other than that, no extra configuration is needed.
```

Comment:
Since you're mentioning some points I suggest you re-format this to an UL format and also remove the last line, since it's a bit of redundant to mention `Other than that, no extra configuration is needed.`.

Something like this would be enough and reads nice in README.

```
There are some things to take keep in mind when working with websockets:

- You cannot bind the guard with `APP_GUARD` or `app.useGlobalGuards()` due to how Nest binds global guards.
- When a limit is reached, Nest will emit an `exception` event, so make sure there is a listener ready for this.
```